config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -4891,6 +4925,40 @@ periodics:
       - --test_args=--ginkgo.focus=\[Flaky\] --ginkgo.skip=\[Feature:.+\] --minStartupPods=8
       env:
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-master
+- tags:
+  - generated   # AUTO-GENERATED by experiment/generate_tests.py - DO NOT EDIT!
+  interval: 6h
+  agent: kubernetes
+  labels:
+    preset-service-account: 'true'
+    preset-k8s-ssh: 'true'
+  name: ci-kubernetes-e2e-gke-ubuntu2-k8sstable3-gpu
+  spec:
+    containers:
+    - args:
+      - --bare
+      - --timeout=200
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=test-5561de90af
+      - --check-leaked-resources
+      - --provider=gke
+      - --deployment=gke
+      - --gcp-zone=us-central1-f
+      - --gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging
+      - --gke-environment=test
+      - --image-family=ubuntu-gke-1604-lts-2
+      - --image-project=ubuntu-os-gke-cloud-devel
+      - --gcp-node-image=custom
+      - --extract=ci/k8s-stable3
+      - --gcp-project-type=gpu-project
+      - --timeout=180m
+      - --test_args=--ginkgo.focus=\[Feature:GPUDevicePlugin\] --minStartupPods=8
+      - --env=NVIDIA_DRIVER_INSTALLER_DAEMONSET=https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/ubuntu/daemonset-preloaded.yaml
```

Comment:
Also, this doesn't exist yet, but Canonical is going to add it, and this will make it easy for them to test.